# f = None
# try:
#     f = open("test.txt", "r")
# except FileNotFoundError as e:
#     # f = open("test.txt", "w")
#     print("文件不存在")
# else:
#     print("哈哈哈，没报错")
# finally:
#     if f != None:
#         f.close()
#
# #  捕获大异常Exception，输出e对象报错内容
# try:
#     print(1/0)
# except Exception as e:
#     print(e)


